src/android/LocalyticsPlugin.java

```diff
@@ -72,21 +72,23 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
     @Override
     public boolean execute(String action, JSONArray args, final CallbackContext callbackContext) throws JSONException {
         if (action.equals("integrate")) {
```

Comment:
i feel like a switch would make this a little easier to read, is there a reason we went with ifs?

Reply:
Java 6 doesn't support switch statements on Strings